3001-4000/LIT3592Miracle.xml

```diff
@@ -33,7 +31,16 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
       </encodingDesc>
       <profileDesc>
          <abstract>
-            <p/>
+            <p>A miracle conflation appears in various manuscripts, linked to this generic miracle
+               theme. To clarify, two CAe IDs have been established. The first mircale version
```

Comment:
you should rephrase the summary to be generic and to point to the two versions through `ref`.